files/virt-v2v-wrapper.py

```diff
@@ -229,6 +229,10 @@ def wrapper(data, state_file, v2v_log):
             '-os', data['export_domain'],
             ])
 
+    if 'network_mappings' in data:
+        for mapping in data['network_mappings']:
+            v2v_args.extend(['--bridge', '%s:%s' % (mapping['source'], mapping['destination'])])
```

Comment:
either check that both `source` and `destination` are in the dict, or use a safer way, e.g.: `mapping.get('source', '')`